packages/network/src/utils.ts

```diff
@@ -75,7 +75,7 @@ export const isValidNetworkConfig = (networkConfig: NetworkConfig | NetworkConfi
       return false
     }
     if (!skipRelayerCheck) {
-      if ((!c.relayerUrl || c.relayerUrl === '') && !c.relayer) {
+      if (!c.relayer || (typeof c.relayer === 'string' && c.relayer === '')) {
```

Comment:
seems the c.relayer will never be of type 'string' ?

Reply:
Good catch!